tests/test-class-wpseo-metabox.php

```diff
@@ -91,6 +91,69 @@ public function test_column_heading_has_metadesc() {
 	}
 
 	/**
+	 * Tests that column_hidden returns the columns if the option isn't set
+	 *
+	 * @covers WPSEO_Metabox::column_hidden()
+	 */
+	public function test_column_hidden_HIDE_COLUMNS() {
+		$user = $this->getMockBuilder( 'WP_User' )
+		             ->getMock();
+
+		// Option may be filled if the user has not set it.
+		$user->expects( $this->any() )
+		     ->method( 'has_prop' )
+		     ->will( $this->returnValue( false ) );
+
+		$expected = array( 'wpseo-title', 'wpseo-metadesc', 'wpseo-focuskw' );
+		$received = self::$class_instance->column_hidden( array(), 'option-name', $user );
+
+		$this->assertEquals( $expected, $received );
+	}
+
+	/**
+	 * Tests that column_hidden returns the option if the option is set
```

Comment:
If I look at your test, I'm assuming you'r testing that wpseo-title is hidden, while the focus keyword and metadesc are not hidden. Could you please elaborate this in your comments so I know what you're testing here?